extra/instrumentData.xml

```diff
@@ -3,146 +3,182 @@
 	<inst name="alf">
 		<location>ndxalf</location>
 		<type>Neutron</type>
+		<displayName>alf</displayName>
```

Comment:
As alluded to previously, all the `displayName`s can be fully capitalised in the first instance, with the exceptions of `SANS2d` and `ChipIr`.